frontend/src/libs/components/input/input.tsx

```diff
@@ -31,9 +35,15 @@ const Input = <T extends FieldValues>({
 
   return (
     <label>
-      <span>{label}</span>
-      <input {...field} type={type} placeholder={placeholder} />
-      {hasError && <span>{error as string}</span>}
+      <span className={styles.inputLabel}>{label}</span>
+      <input
+        {...field}
+        className={`${styles.input} ${hasError ? styles.hasError : ''}`}
```

Comment:
Please check [this comment](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-simpleland/pull/27#discussion_r1135892439) (about multiple/conditional classnames)